src/pages/settings/Wallet/PaymentMethodList.js

```diff
@@ -199,14 +200,23 @@ function PaymentMethodList({
         const paymentCardList = fundList || {};
 
         if (shouldShowAssignedCards) {
-            const assignedCards = _.filter(cardList, (card) => CONST.EXPENSIFY_CARD.ACTIVE_STATES.includes(card.state));
+            const assignedCards = _.chain(cardList)
+                .filter((card) => CONST.EXPENSIFY_CARD.ACTIVE_STATES.includes(card.state))
+                .sortBy((card) => (CardUtils.isExpensifyCard(card.cardID) ? 0 : 1))
+                .value();
+
             return _.map(assignedCards, (card) => {
                 const icon = getBankIcon(card.bank);
+                const isExpensifyCard = CardUtils.isExpensifyCard(card.cardID);
                 return {
-                    key: card.key,
-                    title: translate('walletPage.expensifyCard'),
+                    key: card.cardID,
+                    title: isExpensifyCard ? translate('walletPage.expensifyCard') : card.cardName,
                     description: card.domainName,
-                    onPress: () => Navigation.navigate(ROUTES.SETTINGS_WALLET_DOMAINCARDS.getRoute(card.domainName)),
+                    onPress: isExpensifyCard ? () => Navigation.navigate(ROUTES.SETTINGS_WALLET_DOMAINCARDS.getRoute(card.domainName)) : () => {},
```

Comment:
I think we can keep the `onPress` as is because `interactive` being false should prevent the `onPress` from being triggered, no?